components/console/commands.c

```diff
@@ -240,6 +258,9 @@ static int help_command(int argc, char **argv)
 
 esp_err_t esp_console_register_help_command(void)
 {
+    help_args.help_cmd = arg_str0(NULL, NULL, "<string>", "Name of command");
+    help_args.end = arg_end(1);
+
     esp_console_cmd_t command = {
         .command = "help",
         .help = "Print the list of registered commands",
```

Comment:
Probably we need to add
```
.argtable = &help_args
```
so that the output of `help` describes the `help` command correctly :)